components/Buttons/src/ColorThemer/MDCButtonColorThemer.h

```diff
@@ -61,4 +64,23 @@
 + (void)applyColorScheme:(nonnull id<MDCColorScheme>)colorScheme
                 toButton:(nonnull MDCButton *)button;
 
+/**
+ Applies a color scheme to theme to an MDCFlatButton.
+
+ @param colorScheme The color scheme to apply to @c flatButton.
+ @param flatButton An MDCFlatButton instance to apply a color scheme.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                    toFlatButton:(nonnull MDCButton *)flatButton;
+
+/**
+ Applies a color scheme to theme to an MDCRaisedButton.
+
```

Comment:
Include a link to the new, recommended API.

Reply:
Done.